crates/configuration/src/shared/resources.rs

```diff
@@ -84,6 +108,7 @@ impl ResourcesBuilder {
         Self { config, errors }
     }
 
+    /// Set the memory limit for requests.
```

Comment:
```suggestion
    /// Set the requested memory for the pod. This is the minimum memory that the pod allocate for using.
```

Reply:
I rewrited your doc as: ```Set the requested memory for a pod. This is the minimum memory allocated for a pod.``` wdyt ?

Reply:
Additionally, I updated the limit_(cpu|memory) doc to: ```Set the overall memory limit for a pod. This is the maximum memory threshold for a pod.```

Reply:
Yes, sounds good to me.  Thx!